apps/election-manager/src/utils/ballotLayout.ts

```diff
@@ -0,0 +1,8 @@
+import { BallotPaperSize, Election } from '@votingworks/types'
+
+export const getBallotLayoutPageSize = (election: Election): string =>
+  (election.ballotLayout?.paperSize || BallotPaperSize.Letter).toLowerCase()
+
+export default {
+  getBallotLayoutPageSize,
+}
```

Comment:
Is this useful?

Reply:
Not necessary. Thought some default export was necessary for linter happiness. Will remove. 👍